src/lib.rs

```diff
@@ -1037,15 +1037,21 @@ pub mod pallet {
 		///
 		/// - `query_id`: Query identifier being disputed.
 		/// - `timestamp`: Timestamp being disputed.
+		/// - 'beneficiary`: beneficiary address of user to receive the slash winning
```

Comment:
I think the description should be 'address on controller chain to potentially receive the slash amount if dispute successful' to be a bit more explicit please.